### PR TITLE
cv entity created

### PR DESCRIPTION
**Pull request by jbartek87**

### CODE REVIEW

File: src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Data
+@Table(name="CV")
```
Comment (Crycald) [inline src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java:14]: CV_FILES

File: src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+public class Cv {
```
Comment (Crycald) [inline src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java:15]: Rename class to CvFile

File: src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java (hunk 1 above)
Comment (Crycald) [inline src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java:14]: Rename table name to CV_FILES

File: src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java (hunk 2 above)
Comment (jbartek87) [reply src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java:15]: done

File: src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java (hunk 1 above)
Comment (jbartek87) [reply src/main/java/com/hrsolutionsystem/hrss/model/domain/Cv.java:14]: done

**Comment by jbartek87** [comment]
shut it down

**Pull request closed without merging**